### PR TITLE
rename "add_package_checks" to "do_package_checks"

### PR DESCRIPTION
**Pull request by pat-s**
I don't think we need to specifically deprecate `add_package_checks()` at the current state of the package?

### CODE REVIEW

**Comment by krlmlr** [comment]
Deploy preview for *wonderful-volhard-261e08* ready!

Built with commit b715094b475f591116a159ea51e842c8bb5f8cf3

https://deploy-preview-128--wonderful-volhard-261e08.netlify.com

**Comment by krlmlr** [comment]
https://github.com/search?q=add_package_checks&type=Code finds 117 hits. I'd rather keep a compat version around for some time.

File: R/dsl.R [hunk 1]
```diff
@@ -136,7 +137,55 @@ add_package_checks <- function(...,
 #' @importFrom magrittr %>%
 DSL <- R6Class(
   "DSL",
+  public = list(
```
Comment (krlmlr) [inline R/dsl.R]: Looks like the definition of the `DSL` class is now duplicated.

File: R/dsl.R [hunk 2]
```diff
@@ -136,7 +137,55 @@ add_package_checks <- function(...,
...
+  )
+)
+
+add_package_checks <- function(...,
+                               warnings_are_errors = NULL,
+                               notes_are_errors = NULL,
+                               args = c("--no-manual", "--as-cran"),
+                               build_args = "--force", error_on = "warning",
+                               repos = getOption("repos"), timeout = Inf) {
+  .Deprecated("do_package_checks")
```
Comment (krlmlr) [inline R/dsl.R]: We still need to forward the call to `do_package_checks()`?

File: R/dsl.R (hunk 1 above)
Comment (pat-s) [reply R/dsl.R]: c/p error

File: R/dsl.R (hunk 2 above)
Comment (pat-s) [reply R/dsl.R]: Yes, why did I miss that? :roll_eyes:

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**